src/west/manifest-schema.yml

```diff
@@ -121,14 +121,14 @@ mapping:
             type: any
           groups:
             required: false
-            include: groups
+            include: groups-schema
 
-  # If present, an allowlist/blocklist of project groups. Prefix a
-  # group name with "-" to block it. Give a group name as-is to
-  # allow it.
-  groups:
+  # If present, a list of project groups to enable and disable. Prefix
+  # a group name with "-" to disable it. Give a group name as-is to
+  # enable it.
```

Comment:
Didn't this change to `+` prefix? At least an optional one?

Reply:
Yes, it did. I'll send a fix for master but I might be lazy and leave as-is for the release branch; I'm almost done with all the testing and I don't want to have to start over.

Reply:
(Note the docs are correct)